pkg/resources/testdata/TestAcc_TagAssociation/issue1202/main.tf

```diff
@@ -0,0 +1,31 @@
+resource "snowflake_tag" "test" {
+  name     = var.tag_name
+  database = var.database
+  schema   = var.schema
+}
+
+resource "snowflake_table" "test" {
+  name     = var.table_name
+  database = var.database
+  schema   = var.schema
+  column {
+    name = "test_column"
+    type = "VARIANT"
+  }
+}
+
+resource "snowflake_tag_association" "test" {
+  // we need to set the object_identifier to avoid the following error:
+  // provider_test.go:17: err: resource snowflake_tag_association: object_identifier: Optional or Required must be set, not both
```

Comment:
wasn't that error the result of incorrect schema where you had both Optional and Required? https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2534#discussion_r1508623139

Reply:
yes, but also @sfc-gh-jcieslak wanted to not make any breaking changes by having object_identifier be optional (I think it should be deprecated actually). Also there is the fact that object_name wasn't even being used internally. Since no customers are using this, i have reverted to previous behavior